okhttp/src/main/java/okhttp3/internal/http/HttpHeaders.kt

```diff
@@ -232,3 +233,9 @@ fun Response.promisesBody(): Boolean {
 
   return false
 }
+
+@Deprecated(message = "No longer supported", level = DeprecationLevel.ERROR,
+    replaceWith = ReplaceWith(expression = "response.promisesBody()"))
```

Comment:
1) Since we allow Java users to use this deprecated method, it'd be nice for Kotlin developers to use this function for a while as well? i.e. We might downgrade the error level from `ERROR` to `WARNING`?

2) nit: seems like we usually indent for each param? e.g.
```
@Deprecated(
    message = "No longer supported",
    replaceWith = ReplaceWith(expression = "response.promisesBody()"),
    level = DeprecationLevel.ERROR)
```

Reply:
We have chosen to break source compatibility for kotlin users when upgrading

> OkHttp is not source-compatible for Kotlin callers, but upgrading should be automatic thanks to Kotlin’s powerful deprecation features. Most developers should be able to use IntelliJ’s Code Cleanup for a safe and fast upgrade.